sensor/common/image/service_impl.go

```diff
@@ -113,3 +130,30 @@ func (s *serviceImpl) RegisterServiceHandler(context.Context, *runtime.ServeMux,
 func (s *serviceImpl) AuthFuncOverride(ctx context.Context, fullMethodName string) (context.Context, error) {
 	return ctx, idcheck.AdmissionControlOnly().Authorized(ctx, fullMethodName)
 }
+
+func (s *serviceImpl) Notify(e common.SensorComponentEvent) {
+	switch e {
+	case common.SensorComponentEventCentralReachable:
+		s.centralReady.Signal()
+	case common.SensorComponentEventOfflineMode:
+		s.centralReady.Reset()
+	}
+}
+
+func (s *serviceImpl) Start() error {
+	return nil
+}
+
+func (s *serviceImpl) Stop(_ error) {}
+
+func (s *serviceImpl) Capabilities() []centralsensor.SensorCapability {
+	return nil
+}
+
+func (s *serviceImpl) ProcessMessage(_ *central.MsgToSensor) error {
+	return nil
```

Comment:
Idea for the future: I wonder if it makes sense to have a centralized component to handle everything related to image processing? And if this one is a good candidate for it. Because right now it seems like share the image cache instance between this component, `reprocessor` and `detector`. Maybe it would be good to move cache invalidation and image update messages from central to be processed here?